src/libs/PolicyUtils.ts

```diff
@@ -461,6 +461,45 @@ function getSubmitToAccountID(policy: OnyxEntry<Policy>, employeeAccountID: numb
     return getAccountIDsByLogins([employee.submitsTo ?? defaultApprover])[0];
 }
 
+function getSubmitToEmail(policy: OnyxEntry<Policy>, employeeAccountID: number): string {
+    const submitToAccountID = getSubmitToAccountID(policy, employeeAccountID);
+    return getLoginsByAccountIDs([submitToAccountID])[0] ?? '';
+}
+
+function getForwardsToAccount(policy: OnyxEntry<Policy>, employeeEmail: string, reportTotal: number): string {
+    if (!isControlOnAdvancedApprovalMode(policy)) {
+        return '';
+    }
+
+    const employee = policy?.employeeList?.[employeeEmail];
+    if (!employee) {
+        return '';
+    }
+
+    const positiveReportTotal = Math.abs(reportTotal);
+    if (employee.approvalLimit && employee.overLimitForwardsTo && positiveReportTotal > employee.approvalLimit) {
+        return employee.overLimitForwardsTo;
+    }
+    return employee.forwardsTo ?? '';
+}
+
+function getApprovalChain(policy: OnyxEntry<Policy>, employeeAccountID: number, reportTotal: number): string[] {
```

Comment:
Sorry but I thinkkk we should move this to ReportUtils. Why? Because this approval chain is report-dependent. One report will have a different approval chain than another (assuming someone in the policy has `overLimitForwardsTo` configured).

In our backend code, we also have this (approval chain calculation) in `ReportUtils.php` basically

Reply:
Ahh that makes sense!!
Approval chain is not policy dependent. Noobie mistake

Reply:
What about the funcs `getForwardsToAccount()`, `getSubmitToEmail()`, `getSubmitToAccountID()`?

They will be **different** for reports within the same policy. So we should move them too.
Let me know if you disagree

Reply:
> What about the funcs getForwardsToAccount(), getSubmitToEmail(), getSubmitToAccountID()?
> 
> They will be different for reports within the same policy. So we should move them too.
Let me know if you disagree

Hmm in our backend code, those are all in PolicyUtils b/c `submitsTo` and `forwardsTo` are technically policy settings 🤷